$NetBSD: patch-Makefile,v 1.2 2023/08/06 10:14:50 nia Exp $

Allow using the right compilation flags and installation path

Do not invoke git.

--- Makefile.orig	2015-08-21 22:24:54.000000000 +0000
+++ Makefile
@@ -17,11 +17,11 @@
 #CPATH=/usr/local/lib/lua/5.1
 
 # MacOS (tested with 10.6, 10.7, 10.8)
-CFLAGS=-g -Wall -O2
-LDFLAGS=-bundle -undefined dynamic_lookup -lsasl2
-MACOSX_VERSION=10.5
-LUAPATH=/usr/local/share/lua/5.1
-CPATH=/usr/local/lib/lua/5.1
+#CFLAGS=-g -Wall -O2
+#LDFLAGS=-bundle -undefined dynamic_lookup -lsasl2
+#MACOSX_VERSION=10.5
+#LUAPATH=/usr/local/share/lua/5.1
+#CPATH=/usr/local/lib/lua/5.1
 
 #########################################################
 #
@@ -40,6 +40,7 @@ OBJS=cyrussasl.o luaabstract.o context.o
 all: $(TARGET)
 
 install: $(TARGET)
+	mkdir -p $(CPATH)
 	cp $(TARGET) $(CPATH)
 
 clean:
@@ -67,5 +68,4 @@ context.c: context.h
 version:
 	@if ! test -f $(BUILD_VERSION); then echo 0 > $(BUILD_VERSION); fi
 	@echo $$(($$(cat $(BUILD_VERSION)) + 1)) > $(BUILD_VERSION)
-	@if ! test -f $(BRANCH_VERSION); then git log --pretty=oneline -1|cut -c1-8 > $(BRANCH_VERSION); fi
 
